### PR TITLE
Add possibiity to set enemy's HP

### PR DESCRIPTION
**Pull request by jdel**
Add possibiity to set enemy's HP

### CODE REVIEW

File: lib/filter/enemy.js [hunk 1]
```diff
@@ -54,9 +54,10 @@ exports.update = function(json) {
         rows.push(currentEnemy.params[0].disp_name);
         rows.push(currentEnemy.lv);
         rows.push(currentEnemy.uid);
-        rows.push(currentEnemy.init_hp + '/' + currentEnemy.max_hp);
 
         for (var param in json[round].enemy[enemy].children[part].params) {
+          currentEnemy.init_hp = config.get('enemy.stats.hp') || currentEnemy.init_hp;
```
Comment (ThauEx) [inline lib/filter/enemy.js]: I think there is not need, to put this in this loop. We have to set it only once.

**Comment by jdel** [comment]
Yeah sounds right to me, I'm just hacking away toe code with no idea of what the json looks like.

I will change that and give it a go later.

**Comment by KHShadowrunner** [comment]
Double check - as I think this was the cause of the multi-tiered boss fights glitching out after the boss changed once (like when they shift resistances). I can't remember of HP was affected, but I know I had to do a loop in there for some reason or another.

**Comment by jdel** [comment]
Not sure there are multi tiered boss fights in the events at the moment to test with.

**Comment by jdel** [comment]
Looks fine with assigning the HP out of the loop. 

More testing is needed with multi-tiered bosses I guess

**Comment by ThauEx** [comment]
Do you need a release for this change?

**Comment by jdel** [comment]
No, not necessarily.

**Pull request merged**